<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconOverrideRulesDialog</class>
 <widget class="QDialog" name="IconOverrideRulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Icon Override Rules</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>1060</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="launchersNote">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>475</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: &lt;a href=https://github.com/dangvd/ksmoothdock/wiki/Documentation#editLaunchers_dragAndDrop&gt;Drag and drop&lt;/a&gt; is supported.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset theme="arrow-left">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>400</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="windowNameLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Window Name Pattern</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="windowNameRegex">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>400</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="rulePresets">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>215</y>
     <width>350</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>418</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAll">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IconOverrideRulesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IconOverrideRulesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
